ground/src/main/java/com/google/android/ground/ui/map/gms/mog/MogClient.kt

```diff
@@ -112,13 +113,13 @@ class MogClient(val collection: MogCollection) {
    * ranges are consolidated to minimized the number of individual requests required.
    */
   private suspend fun getTileRequestsForPyramid(
-    mogBounds: TileCoordinates,
+    mogCoordinates: TileCoordinates,
```

Comment:
I prefer `mogBounds` to `mogCoordinates` because it makes helps disambiguate the fact that a MOG file is made up of multiple tiles, each with their own tile coordinates.